utopia-vscode-extension/src/extension.ts

```diff
@@ -505,17 +505,18 @@ async function revealRangeIfPossibleInVisibleEditor(boundsInFile: BoundsInFile):
   )
   if (visibleEditor != null) {
     const rangeToReveal = getVSCodeRangeForScrolling(boundsInFile)
-    const alreadySelected = rangesIntersectLinesOnly(visibleEditor.selection, rangeToReveal)
     const alreadyVisible = visibleEditor.visibleRanges.some((r) =>
       r.contains(visibleEditor.selection),
     )
 
-    if (!alreadySelected) {
-      const selectionRange = getVSCodeRange(boundsInFile)
-      visibleEditor.selection = new vscode.Selection(selectionRange.start, selectionRange.start)
+    const selectionRange = getVSCodeRange(boundsInFile)
+    const newSelection = new vscode.Selection(selectionRange.start, selectionRange.start) // selectionRange.end?
```

Comment:
Right now we're setting the code selection to be the very starting point of the element, and so we're using `selectionRange.start` for both the `anchor` and the `active` values here, where the `anchor` is the end of the selection without a cursor, and the `active` is the end of the selection _with_ the cursor. This is the current setup on production.

If we change the `active` value to use `selectionRange.end` the result is that the entire block of code is selected (and the highlighting reflects that), meaning it would look like this when selecting an element in the navigator:
![Screenshot_20211122_162915](https://user-images.githubusercontent.com/1044774/142898722-20806c9c-61a7-484f-9197-550eef19e928.png)